index.js

```diff
@@ -1,3 +1,6 @@
+console.clear();
```

Comment:
This is only useful during development. Same as with `console.log()`, please remove it before creating a PR.
```suggestion

```